src/lava/proc/io/encoder.py

```diff
@@ -58,10 +58,13 @@ class DeltaEncoder(AbstractProcess):
         Shape of the sigma process.
     vth: int or float
         Threshold of the delta encoder.
-    spike_exp: int
+    spike_exp: Optional[int]
         Scaling exponent with base 2 for the spike message.
         Note: This should only be used for fixed point models.
         Default is 0.
+    num_bits: Optional[int]
+        Precision for spike output. It is applied before spike_exp. If None,
+        precision is not enforced. Default is None.
```

Comment:
What does "precision is not enforced" mean? (What will the behavior be?)

Reply:
Means the output will be unbounded.